src/redux/categories/categories.js

```diff
@@ -0,0 +1,17 @@
+const CHECK = 'app/categories/STATUS_CHECKED';
+
+const reducer = (state = {}, action) => {
```

Comment:
- [ ] Just like I suggested above, you need to set the initial state to an array because it's ideal for storing multiple objects. But from your solution, you used an empty object literal. Please update this.

See example:
```js
const reducer = (state = [ ], action) => { 
     // your code goes here
};
```

Reply:
Thanks for the feedback! will implement these changes